ports/net/macathena-discuss/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup		mit-athena discuss 832df353d2b92d024aa78ddd6b8d0bfdea2a641c
+
+name                macathena-discuss
+version             10.0.17
+revision            1
+categories          net
+maintainers         mit.edu:sipb-macathena
+platforms           darwin
+checksums           rmd160  7edb59085db29dbf5bf6d951105c7c53118bacb1 \
+                    sha256  bf2bf008a1bb817c6c77190d09aca99aed413b2d8dd5ef64fbacd2dd191e580b \
+                    size    173879
+
+use_autoconf		yes
+use_autoreconf		yes
+autoreconf.args		-i -f
+pre-configure {
+	# I think autoreconf is supposed to do this with -i, but it won't so I'm doing it manually
```

Comment:
The Debathena package does this with `automake --foreign --add-missing`. Please don't have a local copy of these files.

Reply:
It fails to build using automake because there is no Makefile.am. Is there a way to tell automake not to look for one?